source/documentation/tools/create-a-derived-table/project-structure.md

```diff
@@ -1,66 +1,723 @@
+- All staging models should live in the `staging` domain regardless of business / service area. This is to maintain visibility of the data being used on create-a-derived-table. Access should be given to databases within the staging domain, not the domain itself. 
+
+Below is an overview of the whole create-a-derived-table folder structure. In the following sections we will go through each layer, covering in detail how we expect your project structure to look and our reasoning. 
+
+```shell
 ├── mojap_derived_tables
   ├── dbt_project.yml
   └── models
-      ├── sources
-      │   ├── nomis.yaml
+      ├── sources # source domain
+      │   ├── nomis.yml # model
       │   ├── oasys_prod.yaml
       │   ├── delius_prod.yaml
       │   ├── xhibit_v1.yaml
       │   ...
-      ├── security  # domain
-      │   ├── prison_safety_and_security  # database
-      │   │   ├── marts
-      │   │   │   ├── intermediate
-      │   │   │   │   ├── prison_safety_and_security__inc_ids_grouped.sql  # intermediate table
-      │   │   │   │   ├── prison_safety_and_security__questions_filtered.sql  # intermediate table
-      │   │   │   │   ├── prison_safety_and_security__question_set_joined.sql  # intermediate table
-      │   │   │   │   └── properties.yaml  # intermediate tables property file
-      │   │   │   └── question_answers  # arbitrary grouping
-      │   │   │       ├── prison_safety_and_security__dates_dim.sql  # dimension table
-      │   │   │       ├── prison_safety_and_security__dates_dim.yaml  # table property file
-      │   │   │       ├── prison_safety_and_security__incidents_fct.sql  # fact table
-      │   │   │       ├── prison_safety_and_security__incidents_fct.yaml  # table property file
-      │   │   │       ├── prison_safety_and_security__question_answer_fct.sql  # fact table
-      │   │   │       └── prison_safety_and_security__question_answer_fct.yaml  # table property file
-      │   │   └── staging
-      │   │       ├── prison_safety_and_security__nomis_mod_stg.md  # markdown to be rendered in documentation
-      │   │       ├── prison_safety_and_security__nomis_mod_stg.sql  # staging table
-      │   │       └── prison_safety_and_security__nomis_mod_stg.yaml  # table property file
+      │ 
+      ├── staging  # staging domain
+      │   ├── stg_nomis # database
+      │   │      ├── stg_nomis__stg_docs.md # model
+      │   │      ├── stg_nomis__stg_models.yml
+      │   │      ├── stg_nomis__stg_offender.sql
+      │   │      └── stg_nomis__stg_prison.sql
+      │   └── stg_xhibit
+      │          ├── _stg_xhibit.yml
+      │          ├── stg_xhibit__stg_<model_names>.sql
+      │          └── stg_xhibit__stg_court.sql
+      ├── courts
+      │   ├── courts_int
+      │   │
+      │   ├── courts_datamarts
+      │   │
+      │   └── criminal_courts_derived
```

Comment:
potentially confusing that we have both `courts` and `criminal_courts` here - are the models common between criminal courts & CFT until the 'derived' layer?

Reply:
I think I was originally trying to show that a derived layer could be a more specific purpose, but I think it is confusin gI will just make them all `criminal_courts_...`